src/views/annual-report/2021/annual-report.jsx

```diff
@@ -0,0 +1,2155 @@
+                                        />
+                                        <h4>
+                                            <FormattedMessage id="annualReport.2021.reachNewUsers" />
+                                        </h4>
+                                        <div className="increase bubble">
+                                            <FormattedMessage id="annualReport.2021.reachNewUsersIncrease" />
+                                        </div>
+                                    </div>
+
+                                    <div className="datapoint world">
+                                        <img
+                                            className="world"
+                                            src="/images/annual-report/2021/0_Data Section/World.svg"
+                                            alt={this.props.intl.formatMessage(
+                                                {id: 'annualReport.2021.altWorldVisualization'}
+                                            )}
+                                        />
+                                        <p>
+                                            <FormattedMessage
+                                                id="annualReport.2021.reachScratchAroundTheWorld"
+                                            />
+                                        </p>
+                                    </div>
+                                    <div className="datapoint saudi-arabia">
+                                        <div className="text-content">
+                                            <h4>
+                                                <FormattedMessage
+                                                    id="annualReport.2021.reachSaudiArabiaTitle"
+                                                />
+                                            </h4>
+                                            <p>
+                                                <FormattedMessage
+                                                    id="annualReport.2021.reachSaudiArabiaDescription"
+                                                />
+                                            </p>
+                                        </div>
+                                        <img
+                                            className="world"
+                                            src="/images/annual-report/2021/0_Data Section/Saudi Arabia.svg"
+                                            alt={this.props.intl.formatMessage(
+                                                {id: 'annualReport.2021.altSaudiArabiaVisualization'}
+                                            )}
+                                        />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div
```

Comment:
When possible, we use class names that describe the content-- could you rename "t"?